### PR TITLE
support julia 0.6

### PR DESCRIPTION
**Pull request by bjarthur**
just gets rid of dep warns and upgrades to Base.Test

### CODE REVIEW

**Comment by bjarthur** [comment]
tests fail only on julia 0.4.  i'm not sure there's a way to make it compatible with 0.4, 0.5 and 0.6.  should we just drop support for 0.4?

**Comment by shashi** [comment]
Thanks!! Yes, let's drop support for 0.4!

**Comment by bjarthur** [comment]
0.4 dropped.  all tests pass

File: src/Patchwork.jl [hunk 1]
```diff
@@ -31,10 +31,10 @@ export Node,
        MaybeKey,
        tohtml
 
-typealias MaybeKey @compat Union{(@compat Void), Symbol}
+const MaybeKey = @compat Union{(@compat Void), Symbol}
 
 # A Patchwork node
-abstract Node
+@compat abstract type Node end
```
Comment (tkelman) [inline src/Patchwork.jl:37]: you need to update the Compat minimum version when you use new features like this

**Pull request merged**